workspaces/rstudio/Dockerfile

```diff
@@ -70,7 +71,7 @@ RUN apt-get update && \
     gosu \
     fonts-dejavu \
     # Utilities for convenience debugging this container:
-    less htop vim nano silversearcher-ag zip unzip git cmake curl wget sqlite3 man-db && \
+    less htop vim nano silversearcher-ag zip unzip git cmake curl wget sqlite3 && \
```

Comment:
Ok, so `man-db` moved ahead.  Fine by me.

Reply:
I made this change to mirror our other images. It may not be strictly necessary.